src/main/java/securbank/controller/ExternalUserController.java

```diff
@@ -36,4 +62,60 @@ public String currentUserDetails(Model model) {
 		
         return "external/detail";
     }
+	
+	@GetMapping("/transaction/create")
+	public String newTransactionForm(Model model){
+		model.addAttribute("transaction", new Transaction());
+		logger.info("GET request: Extrernal user transaction creation request");
+		
+		return "transaction/create";
+	}
+	
+	@PostMapping("/transaction/create")
+    public String submitNewTransaction(@ModelAttribute Transaction transaction, BindingResult bindingResult) {
+		logger.info("POST request: Submit transaction");
+		
+		transactionFormValidator.validate(transaction, bindingResult);
+		
+		if(bindingResult.hasErrors()){
+			return "redirect:/";
+		}
+		
+		if(transaction.getType().contentEquals("CREDIT")){
+			if (transactionService.initiateCredit(transaction) == null) {
+				return "redirect:/";
+			}
+		}
+		else {
+			if (transactionService.initiateDebit(transaction) == null) {
+				return "redirect:/";
+			}
+		}
+		
+		return "transaction/create";
+    }
+	
+	@GetMapping("/external/createtransfer")
+	public String newTransferForm(Model model){
+		model.addAttribute("transfer", new Transfer());
+		logger.info("GET request: Extrernal user transfer creation request");
+		return "external/createtransfer";
+	}
+	
+	@PostMapping("external/createtransfer")
+    public String submitNewTransfer(@ModelAttribute Transfer transfer, BindingResult bindingResult) {
+		logger.info("POST request: Submit transaction");
+		
+		transferFormValidator.validate(transfer, bindingResult);
+		
+		/*if(bindingResult.hasErrors()){
```

Comment:
@ayush2k Have disabled validations. Validator form gives errors for Double regex